purchase_lot/models/purchase_order_line.py

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+#
+#  licence AGPL version 3 or later
+#  see licence in __openerp__.py or http://www.gnu.org/licenses/agpl-3.0.txt
+#  Copyright (C) 2022 Akretion (http://www.akretion.com).
+#  @author David BEAL <david.beal@akretion.com>
+#
+##############################################################################
```

Comment:
Could you adapt this licence header to looks like normal OCA ones?

Reply:
Sure